lib/libexfat.c

```diff
@@ -451,26 +475,19 @@ int exfat_set_volume_label(struct exfat_blk_dev *bd,
 		return -1;
 	}
 
-	vol.type = EXFAT_VOLUME;
-	memset(vol.vol_label, 0, sizeof(vol.vol_label));
```

Comment:
I think we need to zero out all of vol, because Reserved field has to be 0.

Reply:
Okay.